### ISSUE TITLE
Missing parent class members in intelliSense suggestion.

### ISSUE THREAD

**Issue by giyyapan** [microsoft/pyright#188]
**Describe the bug**
Having class B extends class A. IntelliSense suggestions of class B's instance's member only shows those from class B. However the analysis works fine (can determine the defination signature of parent class member).

Please see the image below.

![image](https://user-images.githubusercontent.com/3479453/62755783-f0934980-baa7-11e9-8f64-35cd0421ff0d.png)
_method_a from class A not in intellisense suggesions, but it's signature can be found_

**To Reproduce**
Using the code from screenshot, provided below.

**Expected behavior**
IntelliSense suggestions should contain members from parent classes.

**Screenshots or Code**
```
class A:
    def method_a(self):
        pass

class B(A):
    def method_b(self):
        pass

b = B()


b.method_a()
b. # <- intellisense missing method_a
```

**VS Code extension or command-line**
Using VSCode extension 1.0.53

**Additional context**
I'm using only pyright's intelliSense feature now. It's already much better than Jedi and python language server, great work!

And am I submitting too much issues these days? :rofl:

**Comment by giyyapan** [comment]
BTW, is it better to put those double-underscroe suggestions on the bottom instead of top? It's rarely used in my experience.

**Comment by erictraut** [comment]
Thanks, great find! Please keep the great bug reports and suggestions coming.

This will be fixed in the next version.

**Comment by erictraut** [comment]
This is now fixed in 1.0.54, which I just published.

**Comment by giyyapan** [comment]
Fix confirmed, thanks!
